src/PermMatrix.jl

```diff
@@ -44,6 +44,9 @@ function PermMatrix(
     PermMatrix{Tv,Ti,Vv,Vi}(perm, vals)
 end
 
+Base.:(==)(d1::PermMatrix, d2::PermMatrix) = SparseMatrixCSC(d1) == SparseMatrixCSC(d2)
```

Comment:
Why can't you compare the value directly? Why covert it to CSC?

Reply:
If there are two entries being zero, then their order are not important. The logic is too complicated.